src/dodal/devices/i24/beam_center.py

```diff
@@ -0,0 +1,12 @@
+"""A small temporary device to get the beam center positions from \
+    eiger or pilatus detector on i24"""
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.core import epics_signal_rw
+
+
+class DetectorBeamCenter(StandardReadable):
```

Comment:
I wonder why `StandardDetector` doesn't include signals for beam center. It would be nicer if the detector devices gave us this for free, but this is good in the meantime

Reply:
I think it might be because it just has the more general stuff but not the plug-ins, and the beam center is often in `ancillary data` for the detector?